frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -539,16 +541,30 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
             </FormSelect>
           </FormRow>
           {source.requiresStorageClass() && (
-            <K8sResourceSelectRow
+            <FormSelect
```

Comment:
This change works but you lose the value that comes with `K8sResourceSelectRow` which is the loading state.
I suggest adding a generic `defaultAnnotation` prop to `K8sResourceSelectRow` and update the component to look for it in the annotations and if its there add `(default)` to the name

Reply:
:+1: done